onboarding.md

```diff
@@ -1,279 +1,289 @@
+Lab members must follow the Penn Participation Agreement and the agreements with our sponsors.
+These often allow, encourage, or require openness.
+If you have concerns at any point, set up a meeting with Casey to discuss these concerns.
+
+**Calendars:** There are two Google Calendars for the lab:
+Greene Lab Core Events ([webview](https://calendar.google.com/calendar/embed?src=h1eia9g7qu1udm079vsav7qlq0%40group.calendar.google.com&ctz=America%2FNew_York), Calendar ID `h1eia9g7qu1udm079vsav7qlq0`)
+and Greene Lab Attendance ([webview](https://calendar.google.com/calendar/embed?src=dk2vdln8ci4mh1m723df6rcb3s%40group.calendar.google.com&ctz=America%2FNew_York), Calendar ID `dk2vdln8ci4mh1m723df6rcb3s`).
+The Attendance calendar is for noting individual availability (i.e. whether you'll be out of office).
+It should be used, for example, to note vacations, conference travel, and other workday conflicts.
+All other events should go in the Core Events calendar.
+In general, this calendar is for events that could possibly involve 3 or more lab members.
+Mandatory events such as lab meetings, scrums, and group deadlines go on Core Events.
+
+**Accounts:** Lab members are expected to have accounts for the following and be members of the specified (organizations) if applicable:
+
++ GitHub (greenelab)
++ Google Calender (Shared Calendar)
++ Slack (GreeneLab)
+
+## Meetings
+
+**Scrum:** Our team's scrum process involves three components:
+
+    1. An opening scrum meeting at 9:45 AM Monday morning where individuals will lay out their goals for the week, in the form of post-it notes, on the whiteboard in the Greene Lab.
+    2. A closing scrum meeting at 1:40 PM Friday afternoon where team members will reflect on the progress they made on the goals they stated the previous Monday.
+    3. A daily virtual scrum update.
+
+The daily virtual scrum update should include an update to [scrum repository](https://github.com/greenelab/scrum).
+An issue is automatically created for each day the office is open.
+These issues can be found [here](https://github.com/greenelab/scrum/issues).
+The update should include the following:
+
+    1. What specific item(s) he/she accomplished yesterday.
+    2. What specific item(s) he/she plans to accomplish today.
+    3. Who, if anyone, is blocking him/her?
+    4. Who, if anyone, is he/she blocking?
+
+**Lab Meeting:** Lab meeting is held weekly at a location at Penn and also virtually via [uberconference](https://uberconference.com/greenescientist).
+Scheduling is managed via a google spreadsheet.
+See the `#general` slack channel's pinned items link.
+Lab meeting consists of two components described below.
+
+- Journal Club
+- Braintrust
+
+**Journal Club:** This is an opportunity for lab members to strengthen their ability to comprehend and critically analyze research that has already been conducted.
+For journal club, everyone will be randomly split off into groups of two (or three if an odd number of lab members).

```

Comment:
Is this like "think-pair-share" (zoom break out rooms) or is it like "presenting in teams"? I guess I am just a little unclear on what people are supposed to be prepping -- would it make sense to explain how/when the 4 papers are selected and when the groups are determined?

Reply:
I was under the impression that the teams would be known at least a week or two in advance of the relevant meeting (so more like "presenting in teams"), and that people that are in a team together would then correspond prior to the meeting to decide on the papers and put together a presentation. Having something prepared in advance seems to me like the best use of the hour-long meeting.

But I'm curious to hear what other people think as well.

Reply:
@jjc2718 is correct. Basically groups would be assigned once the semester lab schedule has been finalized. So everyone would have plenty of time to correspond with their group member(s) on papers and topics to be discussed. If different idea is desired feel free to suggest changes or pitch your idea here.

Reply:
```suggestion
At the beginning of each semester, everyone will be randomly spli into groups of two (or three if an odd number of lab members) for journal club.

```